### DISCUSSION TITLE
Error with build cache on Digital Ocean

### DISCUSSION THREAD

**Discussion by ScreamZ** [vercel/turborepo#850] [Help] [answered]
Hello,

**🙋🏻‍♂️ Temporary workaround:** For now I've been able to fix it by doing like so 
```
cd apps/backend
yarn
cd ../../
npx turbo run build --scope=@easy-inscription/backend --include-dependencies --no-deps
```
---

I'm trying to deploy a NestJS backend server on a digital ocean droplet using DigitalOcean Apps.

Here is my configuration, based on something like the one recommended for NextJS on Vercel.
<img width="1020" alt="Screenshot 2022-03-09 at 09 41 31" src="https://user-images.githubusercontent.com/6640835/157404674-03ade199-eaf1-467d-8f4a-fe5461fcc874.png">

I'm not sure about the `--no-deps` and `--include-dependencies`, my first question is can someone give me some intel about it?

This works well on the first build but then digital ocean seems to cache `node_modules` and fail because `@nestjs` deps are in the apps/backend folder and not in the root folder which seems to be the only one cached
<img width="1363" alt="Screenshot 2022-03-09 at 09 30 34" src="https://user-images.githubusercontent.com/6640835/157403805-db1b5415-d559-45ec-a84c-89cb08ba5423.png">

**Comment by alexandersazonof** [comment]
Hi, I have the same problem. I resolved this just remove node_modules
```
rm -rf node_modules && yarn install && yarn build
```

**Comment by anthonyshew** [comment, marked as answer]
Sounds like you sorted out your answer in original post!
